game/src/assets/html/settings.html

```diff
@@ -1,5 +1,5 @@
-<h1>Coming soon...</h1>
-
+<h1>Settings</h1>
+<input data-value id="settingsCheck" type="checkbox" > Enable Offline Mode
```

Comment:
`data-value` should contain a value. `swCheck` I think is the name you are using.